drivers/clock_control/Kconfig.stm32f0_f3

```diff
@@ -7,23 +7,25 @@ if SOC_SERIES_STM32F0X || SOC_SERIES_STM32F3X
 
 config CLOCK_STM32_PLL_PREDIV
 	int "PREDIV Prescaler"
+	depends on CLOCK_STM32_SYSCLK_SRC_PLL
 	default 1
 	range 1 16
 	help
-	  PREDIV is PLLSCR clock signal prescaler, allowed values: 1 - 16.
+	  PREDIV is a PLL clock signal prescaler for the HSE output.
```

Comment:
so why removing  "depends on CLOCK_STM32_PLL_SRC_HSE"

Reply:
I'm removing it for CLOCK_STM32_PLL_PREDIV1 where it shouldn't be. There was no such dependency for CLOCK_STM32_PLL_PREDIV, and I'm not sure I should introduce it now.

Reply:
ok